docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.2'
+services:
+  start_services:
```

Comment:
Could we document the usage of this in the README as well, please?

Reply:
Good point!